### PR TITLE
CI/CD Setup for Mainnet Deployments

### PR DESCRIPTION
**Pull request by systemblox**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -129,12 +129,22 @@ Deploy nodes to prod:
     # |  Deploy SSV nodes  |
```
Comment (olegshmuelov) [inline .gitlab-ci.yml:129]: Change to Deploy SSV Prater nodes

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -129,12 +129,22 @@ Deploy nodes to prod:
...
     # +--------------------+
     - .k8/production/prater/scripts/deploy-cluster-1--4.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
+    # +---------------------------+
+    # | Deploy SSV Mainnet nodes  |
+    # +---------------------------+
+    # - .k8/production/mainnet/scripts/deploy-cluster-1-4.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
     #
     # +-------------------+
     # │  Deploy Bootnode  |
```
Comment (olegshmuelov) [inline .gitlab-ci.yml:138]: change to Deploy Prater Bootnode

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -153,5 +163,10 @@ Deploy exporter to prod:
     - chmod +x ./kubectl
     - mv ./kubectl /usr/bin/kubectl
     - .k8/production/prater/scripts/deploy-exporters.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $SSV_EXPORTER_CPU_LIMIT $SSV_EXPORTER_MEM_LIMIT
+    # +---------------------------+
+    # │  Deploy Mainnet exporter  |
+    # +---------------------------+
```
Comment (olegshmuelov) [inline .gitlab-ci.yml:168]: Please add the same for Prater explorer

File: .k8/production/mainnet/ssv-exporter-1.yml [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+          - name: NETWORK
+            value: "mainnet"
+          - name: DB_PATH
+            value: "./data/db-mainnet"
+          - name: METRICS_API_PORT
+            value: "15015"
+          - name: ENABLE_PROFILE
+            value: "true"
+          - name: GENERATE_OPERATOR_KEY
+            value: "true"
```
Comment (olegshmuelov) [inline .k8/production/mainnet/ssv-exporter-1.yml:139]: should be removed

File: .k8/production/mainnet/ssv-exporter-2.yml [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+          - name: NETWORK
+            value: "mainnet"
+          - name: DB_PATH
+            value: "./data/db-mainnet"
+          - name: METRICS_API_PORT
+            value: "15016"
+          - name: ENABLE_PROFILE
+            value: "true"
+          - name: GENERATE_OPERATOR_KEY
+            value: "true"
```
Comment (olegshmuelov) [inline .k8/production/mainnet/ssv-exporter-2.yml:139]: should be removed

**Review by olegshmuelov — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (systemblox) [reply .gitlab-ci.yml]: done

File: .k8/production/mainnet/ssv-exporter-1.yml (hunk 4 above)
Comment (systemblox) [reply .k8/production/mainnet/ssv-exporter-1.yml]: done

File: .gitlab-ci.yml (hunk 2 above)
Comment (systemblox) [reply .gitlab-ci.yml]: done

**Review by olegshmuelov — APPROVED** [review]

**Review by moshe-blox — APPROVED** [review]

File: .gitlab-ci.yml [hunk 6]
```diff
@@ -125,16 +126,26 @@ Deploy nodes to prod:
...
+    # +--------------------------+
+    # │  Deploy Prater Bootnode  |
+    # +--------------------------+
     # █▓▒░ Keep commented unless you're testing the bootnode ░▒▓█
     # - .k8/production/prater/scripts/deploy-boot-nodes.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
+    # +---------------------------+
+    # │  Deploy Mainnet Bootnode  |
+    # +---------------------------+
+    # █▓▒░ Keep commented unless you're testing the bootnode ░▒▓█
+    - .k8/production/mainnet/scripts/deploy-boot-nodes.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
```
Comment (lior-blox) [inline .gitlab-ci.yml:147]: @moshe-blox @systemblox this should be commented

File: networkconfig/mainnet.go [hunk 7]
```diff
@@ -12,9 +13,13 @@ var Mainnet = NetworkConfig{
...
-	ETH1SyncOffset:       new(big.Int).SetInt64(8661727),
-	RegistryContractAddr: "", // TODO: set up
-	Bootnodes:            []string{
-		// TODO: fill
+	ETH1SyncOffset:       new(big.Int).SetInt64(17428695),
+	RegistryContractAddr: "0x42Cd8D240E30102B715d7516f97864ECeC4441Ab",
+	Bootnodes: []string{
+		"enr:-Li4QHEPYASj5ZY3BXXKXAoWcoIw0ChgUlTtfOSxgNlYxlmpEWUR_K6Nr04VXsMpWSQxWWM4QHDyypnl92DQNpWkMS-GAYiWUvo8h2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhCzmKVSJc2VjcDI1NmsxoQOW29na1pUAQw4jF3g0zsPgJG89ViHJOOkHFFklnC2UyIN0Y3CCE4qDdWRwgg-i",
+	},
+	WhitelistedOperatorKeys: []string{
```
Comment (lior-blox) [inline networkconfig/mainnet.go:21]: @moshe-blox please add a comment that these are blox exporters in all configs

**Pull request merged**